README.md

```diff
@@ -141,21 +142,46 @@ The service will respond to requests on `localhost:2424`, and the admin data wil
 
 ### Configuration
 
-Configuration is handled by [Viper](https://github.com/spf13/viper#putting-values-into-viper).
-The easiest way to set config during development is by editing the [config.yaml](./config.yaml) file.
+Configuration is handled by [Viper](https://github.com/spf13/viper#putting-values-into-viper). The easiest way to set config during development is by editing the [config.yaml](./config.yaml) file. You can also set the config through environment variables. For instance:
+
+```bash
+export PBC_COMPRESSION_TYPE="none"
+```
+##### Rate limiter configuration
+
+Prebid Cache's rate limiting feature, that has the downside of considerable memory consumption, is enabled by default for a maximum of 100 requests per second. From the [config.yaml](./config.yaml) file, use the `rate_limiter.enabled` and `rate_limiter.num_requests` options to either disable the rate limiter or modify its request capacity. For instance:
+
+```yaml
+rate_limiter:
+  enabled: false
+```
 
-You can also set the config through environment variables. For example:
+disables the rate limiter. We could also disable setting the following environment variable:
+
+```bash
+export PBC_RATE_LIMITER_ENABLED="false"
+```
+
+In contrast, we could keep the rate limiter running and set its maximum number of requests to a value other than 100. Inside [config.yaml](./config.yaml):
+
+```yaml
+rate_limiter:
+  num_requests: 150
+```
 
+Or via the following environment variable:
 ```bash
-PBC_COMPRESSION_TYPE=none ./prebid-cache
+export PBC_RATE_LIMITER_NUM_REQUESTS=150
 ```
 
 ### Docker
 
 Prebid Cache works in Docker out of the box.
 
 ```bash
-CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo  .
+export CGO_ENABLED=0
```

Comment:
I know this is not your change but I am wondering if we even need these environment variables to set? We're building Prebid Cache in a docker container and I believe these should be set in the container because I don't think there will be any effect of them having set outside the container

Reply:
Done